### ISSUE TITLE
GIFs aren't animated

### ISSUE THREAD

**Issue by Aariq** [geerlingguy/pi-timelapse#23]
First off, thank you for making this script! I'm so excited to get this project going.

I was able to follow your tutorial and run timelapse.py, and I see that it produced a number of images and a gif file, but the .gif seems like it's just a single image. When I run the timelapse.py script, the message `Creating animated gif` appears _before_ the `Time-lapse capture complete!` message.

I'm using raspian buster lite on a pi zero w.

**Comment by Aariq** [comment]
Update: if I switch to using a for loop instead of the threading it works like expected.

**Comment by geerlingguy** [comment]
Yeah; the gif creation might or might not be working as-is right now. I originally created that in the first iteration of the script, but didn't test it much after I made the changes to use threading... the problem is I'm now mostly using it for very long timelapses (which would take literally days to render into a gif on my poor Pi Zero W), so I haven't really needed to generate gifs anymore.
